src/client/components/definition/definitionData.js

```diff
@@ -39,16 +39,17 @@ module.exports = {
       {
         title: 'Price-Earnings Ratio (EPS)',
         definition: 'Used for valuing a company, P/E ratio the ratio of ' +
-        'a company\'s share price to it\'s earnings per share. ' +
+        'a company\'s share price to its earnings per share. ' +
```

Comment:
huh, i thought i addressed this in a previous PR...

Reply:
I though so too.  Maybe there was a similar mistake in a multiple places?